.jenkins/scripts/global/init-access.sh

```diff
@@ -0,0 +1,10 @@
+echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ bionic main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
+wget https://packages.microsoft.com/keys/microsoft.asc
+sudo apt-key add microsoft.asc
+sudo apt-get update
+while sudo lsof /var/lib/dpkg/lock-frontend | grep dpkg; do sleep 3; done
```

Comment:
I've been thinking if this should load the OE library just to use `helpers.WaitForAptLock()` as this only covers one potential lock.

Reply:
The functionality of this file was already split into different scripts, I forgot to update this one.